src/utils/test.js

```diff
@@ -40,13 +40,13 @@ export { userEvent };
 
 // throw if console.error is called in tests
 beforeEach(() => {
-  jest.spyOn(console, "error").mockImplementation(() => {
-    throw new Error();
-  });
+  // jest.spyOn(console, "error").mockImplementation(() => {
```

Comment:
![why](https://user-images.githubusercontent.com/12068482/119764428-b309e580-bef4-11eb-8c7f-a096b5a321c0.gif)

Reply:
Because anytime the console has an error it fails the test without actually showing what the error was. Which is the most infuriating behaviour. I will try and see if I can have the test fail without actually mocking the console.error

Reply:
Because anytime the console has an error it fails the test without actually showing what the error was. Which is the most infuriating behaviour. I will try and see if I can have the test fail without actually mocking the console.error